components/micro-gateway-core/src/main/ballerina/src/gateway/constants/constants.bal

```diff
@@ -122,6 +122,7 @@ public const string USER_IP_PROPERTY = "userIp";
 public const string REQUEST_TIME_PROPERTY = "requestTimestamp";
 public const string GATEWAY_TYPE_PROPERTY = "gatewayType";
 public const string GATEWAY_TYPE = "MICRO";
+public const string ADDITIONAL_PROPS = "ADDITIONAL_PROPS";
```

Comment:
Shall we use some name like "ADDITIONAL_ANALYTICS_PROPS"

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/1319/commits/b694c6960318d34763d343da195320faf623a55f